app/src/main/java/com/example/caregiver/ProfileInfo.java

```diff
@@ -298,6 +298,10 @@ public void onClick(View v) {
     private View.OnClickListener logOutListener = new View.OnClickListener() {
         @Override
         public void onClick(View v) {
+            SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(getContext());
```

Comment:
I add this here to be a double safety check to ensure user information are deleted properly once they log out instead of linger in the app. If we do decide to do "Remember Me", we might have to modify this later to choose which information to store and which to delete.